wikipedia-sentences-incremental/app.py

```diff
@@ -9,51 +9,51 @@
 
 
 def config():
-    os.environ['JINA_DATA_FILE_1'] = os.environ.get(
-        'JINA_DATA_FILE_1', 'data/input-1.txt'
+    os.environ["JINA_DATA_FILE_1"] = os.environ.get(
```

Comment:
I think your editor switched it back to double quotes, can you change it again?

Reply:
Are we using the Black code formatter (I think @maximilianwerk mentioned it a while ago)

I ran `:Black` from vim and it switched it all to double quotes

Reply:
I think we are not using it yet, but the standard with us is to use single-quotes

Reply:
Fixed. Now only place with double quotes is nested quotes (lines 39, 46)